web-share/share-extra-argument-manual.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>WebShare Test: Surplus arguments ignored</title>
```

Comment:
> If that can be solved generically, I think that something in idlharness.js might make sense.

This is what I would have suggested except afaik we don't have a good story for using idlharness on manual tests for now.